heroic-dist/src/test/java/com/spotify/heroic/HeroicDistributionGenerator.java

```diff
@@ -1,45 +1,69 @@
 package com.spotify.heroic;
 
+import static com.google.common.math.Quantiles.percentiles;
+
+
 import com.google.protobuf.ByteString;
 import com.spotify.heroic.metric.HeroicDistribution;
+import com.tdunning.math.stats.MergingDigest;
 import com.tdunning.math.stats.TDigest;
 import java.nio.ByteBuffer;
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import org.apache.commons.math3.distribution.ParetoDistribution;
 
 
 public class HeroicDistributionGenerator {
 
-  static HeroicDistribution generateOne(){
-       List<Double> data = Data.generateParetoDistribution(1000);
-       TDigest tDigest =TDigest.createMergingDigest(100);
-       recordValue(tDigest, data);
-       return HeroicDistribution.create(encode(tDigest));
+    static HeroicDistribution generateOne(){
```

Comment:
i think this might've been spaced in too much

Reply:
This is probably fine.